### PR TITLE
Made the adding cover image feature visible in mobile devices when there is no cover image

### LINKED ISSUE #4146: When there is no cover image of a facility you cannnot add the cover image in mobile devices
**Describe the bug**
When there is no cover image of a facility you cannnot add the cover image in mobile devices

**To Reproduce**
Go to any facility and delete the facility cover image and view it in mobile device 

**Expected behavior**
You should be able to add the cover image from mobile devices also.

**Screenshots**
image

### PR DESCRIPTION
**Pull request by patelaryan7751**
## Proposed Changes

- Fixes #4146 
Made the adding cover image feature in mobile devices when there is no cover image
image
image

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Facility/FacilityHome.tsx [hunk 1]
```diff
@@ -302,6 +302,21 @@ export const FacilityHome = (props: any) => {
           <CoverImage />
           {editCoverImageTooltip}
         </div>
+      ) : (
+        <div
+          className={`group md:hidden flex w-full self-stretch shrink-0 bg-gray-300 items-center justify-center relative z-0 ${
+            hasPermissionToEditCoverImage && "cursor-pointer"
+          }`}
```
Comment (developedBySJ) [inline src/Components/Facility/FacilityHome.tsx:310]: Use `clsx`

**Review by developedBySJ — CHANGES_REQUESTED** [review]
For Desktop devices, User can see the upload button on hover but for mobile, It is hard to find it
Show the upload button by default for mobile devices

**Comment by nihal467** [comment]
@patelaryan7751 i can see a comment by @developedBySJ once its complete, mark it for testing

File: src/Components/Facility/FacilityHome.tsx (hunk 1 above)
Comment (patelaryan7751) [reply src/Components/Facility/FacilityHome.tsx:310]: Hey @developedBySJ Gigin has told no to use clsx totally. If we show the pencil icon by default in Mobile Devices the cover image would not be visible in mobile devices.

**Comment by nihal467** [comment]
LGTM

File: src/Components/Facility/FacilityHome.tsx (hunk 1 above)
Comment (khavinshankar) [reply src/Components/Facility/FacilityHome.tsx:310]: @patelaryan7751 can you refactor it using classNames _(should be available in `src/Utils/utils.ts`)_

**Review by khavinshankar — CHANGES_REQUESTED** [review]

**Pull request merged**